AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/SQLStatement+Condition.swift

```diff
@@ -32,7 +32,7 @@ private func translateQueryPredicate(from modelSchema: ModelSchema,
         let indent = String(repeating: indentPrefix, count: indentSize)
         if let operation = pred as? QueryPredicateOperation {
             let logicalOperator = groupOpened ? "" : "\(groupType.rawValue) "
-            let column = operation.operator.columnFor(field: operation.field,
```

Comment:
what type is `operation.field`? can we build an extension method on `operation.field` in AWSPluginsCore like
```swift
func getColumnName(_ modelSchema: ModelSchema) {
        switch modelSchema.field(withName: operation.field)?.association {
        case .belongsTo(_, let targetName):
            return targetName ?? self
        default:
            break
        }
        return self
}
```
then you can use it from both DataStore and API